lib/typecheck.js

```diff
@@ -607,9 +607,21 @@ function loadAllSchemas(ast, schemas, scope, classes, useMeta, isMeta = false) {
     }));
 }
 
-async function typeCheckClass(klass, schemas) {
+async function typeCheckClass(klass, schemas, isMeta = false) {
+    if (isMeta) {
+        Object.entries(klass.metadata).forEach(([name, value]) => {
+            if (typeof value !== 'string')
+                throw new TypeError('Invalid value type for natural language annotations');
+        });
+    }
+    let imported = new Set();
     for (let import_stmt of klass.imports) {
         if (import_stmt.isMixin) {
+            for (let facet of import_stmt.facets) {
+                if (imported.has(facet))
+                    throw new TypeError(`Multiple ${facet} are imported`);
```

Comment:
How about "`config` mixin imported multiple times" - I don't quite understand the error message otherwise